frontend/indigenousapprenticeship/package.json

```diff
@@ -109,8 +109,10 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@heroicons/react": "^2.0.10",
-    "core-js": "^3.25.0",
+    "@heroicons/react": "^1.0.6",
```

Comment:
Careful, we don't want to roll back these two dependencies.  Probably a merge-main artifact.